### PR TITLE
Jsl mx98360a

### PR DESCRIPTION
**Pull request by udaymb**
Added support for ma98360a codec on existing jsl-rt5682 topology. 
Separated topology build configuration for sof-jsl-rt5682-rt1015 & sof-jsl-rt5682-mx98360a

### CODE REVIEW

**Comment by lgirdwood** [comment]
@udaymb can you check CI build.

File: tools/topology/sof-jsl-rt5682.m4 [hunk 1]
```diff
@@ -16,8 +16,7 @@ include(`common/tlv.m4')
 include(`sof/tokens.m4')
 
 # Include JasperLake DSP configuration
-include(`platform/intel/icl.m4')
-include(`platform/intel/dmic.m4')
```
[the remark this answers is not in the document]
Comment (udaymb) [reply tools/topology/sof-jsl-rt5682.m4:20]: dmic.m4 included in icl.m4. 
https://github.com/thesofproject/sof/blob/700ca2ae9a0c27a0e2e201de0fa358a58d4a5810/tools/topology/platform/intel/icl.m4#L84

File: tools/topology/platform/intel/jsl-rt1015.m4 [hunk 2]
```diff
@@ -0,0 +1,12 @@
...
+#
+
+include(`platform/intel/icl.m4')
+
+define(`SET_SSP_CONFIG',
+				`SSP_CONFIG(I2S, SSP_CLOCK(mclk, 24000000, codec_mclk_in),
+						SSP_CLOCK(bclk, 2400000, codec_slave),
+						SSP_CLOCK(fsync, 48000, codec_slave),
+						SSP_TDM(2, 25, 3, 3),
+						SSP_CONFIG_DATA(SSP, 1, 24))')
```
[the remark this answers is not in the document]
Comment (udaymb) [reply tools/topology/platform/intel/jsl-rt1015.m4:18]: Done.
platform/intel/jsl-rt1015.m4 file changes (Included macros) are similar to platform/intel/jsl-dedede.m4.

**Comment by yongzhi1** [comment]
cc @cujomalainey @bzhg here is the topology change for drawman, thanks!!

**Comment by lgirdwood** [comment]
jenkins showing known issues.

**Pull request merged**